### ISSUE TITLE
Strava now has an RPE field, but this does not carry over to GC when synchronizing. Can this be fixed/added?

### ISSUE THREAD

**Issue by nitishnag** [GoldenCheetah/GoldenCheetah#3444]
Issue tracker is **only** for Bugs and Features, before to open a new issue please read the contributing guidelines (link below), and use the forums if you need help or technical support.

**Comment by liversedge** [comment]
No, there is no API to retreive it or set it from outside Strava.
This has been investigated previously. Will put into the parking lot.
